asciidoc/get-started.adoc

```diff
@@ -379,14 +379,16 @@ include::{python-examples}/test_driver_introduction_example.py[tags=driver-intro
 [[driver-api-docs]]
 == Driver API docs
 
+:driver-version: 4.2
```

Comment:
This isn't being used because the attribute is already set in antora.yml but should be removed because if it was used it would output the wrong value.

Reply:
Ahh, didn't spot the antora file in this one until now, only just realizing now I didn't even set the right version...